bot/exts/moderation/watchchannels/talentpool.py

```diff
@@ -176,13 +175,39 @@ async def nomination_edit_group(self, ctx: Context) -> None:
 
     @nomination_edit_group.command(name='reason')
     @has_any_role(*MODERATION_ROLES)
-    async def edit_reason_command(self, ctx: Context, nomination_id: int, *, reason: str) -> None:
-        """
-        Edits the reason/unnominate reason for the nomination with the given `id` depending on the status.
+    async def edit_reason_command(self, ctx: Context, nomination_id: int, actor: FetchedMember, *, reason: str) -> None:
+        """Edits the reason of `actor` entry for the nomination with the given `id`."""
```

Comment:
The wording here is a little confusing, I'd go for something like:
```suggestion
        """Edits the reason of a specific nominator in a specific active nomination."""
```